akka-docs-dev/rst/migration-guide-1.0-2.x.rst

```diff
@@ -254,10 +396,48 @@ Update procedure
 
 We show the necessary steps in terms of an example ``AsyncStage``
```

Comment:
Remove this sentence or replace with explanation that we've replaced it with GraphStage